extensions/shared/upgrade-nudge/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { __, sprintf } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { Button } from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import getSiteFragment from '../get-site-fragment';
+import './store';
+
+import './style.scss';
+
+const UpgradeNudge = ( { feature, plan, planName } ) => (
+	<div className="upgrade-nudge">
+		<Gridicon className="upgrade-nudge__icon" icon="star" size={ 18 } />
+		<div className="upgrade-nudge__info">
+			<span className="upgrade-nudge__title">
+				{ sprintf( __( 'Upgrade to %(planName)s', 'jetpack' ), {
+					planName,
+				} ) }
+			</span>
+			<span className="upgrade-nudge__message">
+				{ __( 'To make this block visible on your site' ) }
```

Comment:
Missing `jetpack` textdomain here and on line 40 for "Upgrade".

Reply:
Good spot! Fixed in 07bf211.